apps/web/app/s/[surveyId]/components/LinkSurvey.tsx

```diff
@@ -181,7 +181,9 @@ export default function LinkSurvey({
 
   return (
     <div className="flex h-screen items-center justify-center">
-      {!determineStyling().isLogoHidden && product.logo?.url && <ClientLogo product={product} />}
+      {product.logo?.url && (
+        <ClientLogo product={product} isLogoHidden={determineStyling().isLogoHidden ?? true} />
```

Comment:
I think this logic doesn't play out right. It should only be hidden if it is clearly set to hidden and not vice versa. If anything, it should be "?? false" so that it only gets hidden, when its definitely set to hidden? 

Not entirely sure, havent worked with "??" yet but have a look at this video:

https://github.com/formbricks/formbricks/assets/72809645/1dd31120-77ad-45f5-9231-c0edbca5537e